spec/apiForSheetsSpec.js

```diff
@@ -59,6 +59,10 @@ describe('Api For Sheets Module', () => {
           data: commitsData
         }),
       },
+      repos: {
+        checkCollaborator: jasmine.createSpy(
+          'checkCollaborator').and.returnValue(true),
+      }
```

Comment:
I'm not sure why this has been added here.

Reply:
Updated - created spy calls instead of this.